### PR TITLE
Release 0.4.0

### PR DESCRIPTION
**Pull request by SilasKenneth**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
     }
   },
   "require": {
-    "microsoft/kiota-abstractions": "^0.4.0",
+    "microsoft/kiota-abstractions": "^0.5.0",
```
Comment (SilasKenneth) [inline composer.json:19]:
```suggestion
    "microsoft/kiota-abstractions": "^0.6.0",
```

Comment (Ndiritu) [reply composer.json:19]: Unable to commit the suggestion. I think it works if I suggest the change then you commit

Comment (Ndiritu) [inline composer.json:19]:
```suggestion
    "microsoft/kiota-abstractions": "^0.6.0",
```

**Review by Ndiritu — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (SilasKenneth) [reply composer.json:19]: It's the same issue. I created a PR for it.

**Review by Ndiritu — APPROVED** [review]

**Pull request merged**